js/adapt-launchNewWindow.js

```diff
@@ -1,225 +1,225 @@
+        window._relaunched();
+      }
 
-            this.stopAdaptLoading();
-            this.newLaunchView();
+      if (!this.isEnabled()) {
+        // If the window has been relaunched then just proceed as normal loading Adapt
+        Adapt.trigger('offlineStorage:prepare');
+        Adapt.config.set('_canTriggerDataLoaded', true);
+        return;
+      }
 
-        },
+      this.stopAdaptLoading();
+      this.newLaunchView();
 
-        isEnabled: function() {
+    },
 
-            // Config not defined or enabled
-            if (!this._config || !this._config._isEnabled) return;
+    isEnabled: function() {
 
-            // Check relaunched in search part incase window.open doesn't work properly
-            if (/relaunched=y/.test(location.search)) return;
+      // Config not defined or enabled
+      if (!this._config || !this._config._isEnabled) return;
 
-            // Check if active on this page
-            if (this._config._activeOnPages && this._config._activeOnPages.length > 0) {
-                var found = false;
-                this._config._activeOnPages.forEach(function(pageName) {
-                    if (new RegExp(pageName).test(location.href)) {
-                        found = true;
-                    }
-                });
-                if (!found) return;
-            }
+      // Check relaunched in search part incase window.open doesn't work properly
+      if (/relaunched=y/.test(location.search)) return;
 
-            // Check if the launch mode matches properly
-            this.mode = this.getLaunchMode();
-            if (this.mode !== LAUNCH_MODE.NEW_WINDOW) return;
+      // Check if active on this page
+      if (this._config._activeOnPages && this._config._activeOnPages.length > 0) {
+        var found = false;
+        this._config._activeOnPages.forEach(function(pageName) {
+          if (new RegExp(pageName).test(location.href)) {
+            found = true;
+          }
+        });
+        if (!found) return;
+      }
 
-            this.href = this.getHREF();
-            this.delay = this.getDelay();
+      // Check if the launch mode matches properly
+      this.mode = this.getLaunchMode();
+      if (this.mode !== LAUNCH_MODE.NEW_WINDOW) return;
 
-            return true;
+      this.href = this.getHREF();
+      this.delay = this.getDelay();
 
-        },
+      return true;
 
-        getLaunchMode: function() {
+    },
 
-            var $html = $("html");
-            var isNewWindow = this._config._activeOnSelector && $html.is(this._config._activeOnSelector);
-            
-            if (isNewWindow) return LAUNCH_MODE.NEW_WINDOW;
+    getLaunchMode: function() {
+      if (!this._config._activeOnSelector) return LAUNCH_MODE.NEW_WINDOW;
 
-            return LAUNCH_MODE.CURRENT_WINDOW;
+      var $html = $('html');
+      var isNewWindow = this._config._activeOnSelector && $html.is(this._config._activeOnSelector);
```

Comment:
`this._config._activeOnSelector` will _always_ be true here with the new line 87.

Logic could simply be:
```js
var selector = this._config._activeOnSelector;
var isNewWindow = !selector || $('html').is(selector);
```
(no idea why `$html` was cached).

Reply:
good call. and with this style I think `isNewWindow` is pretty redundant, the name of the ENUM makes it pretty clear what's going on.